src/engine/worker.cpp

```diff
@@ -18,8 +19,59 @@
 #include "sampling/logits_processor.h"
 #include "sampling/sampler.h"
 
+DEFINE_bool(enable_cudagraph, false, "Enable CUDAGraph to optimize model execution.");
+
 namespace llm {
 
+class CudaGraphRunner {
```

Comment:
sounds we can create a simple model to test this cuda graph runner.

Reply:
yep